README.md

```diff
@@ -22,62 +22,38 @@ A GitHub action for styling files with [prettier](https://prettier.io).
 | commit_options | :x: | - | Custom git commit options |
 | commit_message | :x: | Prettified Code! | Custom git commit message |
 | file_pattern | :x: | * | Custom git add file pattern |
-| branch | :white_check_mark: | - | There are two types of action triggers in GitHub: on pull request and on push. The branch needs to be defined for both, but in case of the pull request trigger it should have `${{ github.head_ref }}` and on push it should have the branch the trigger is designed for. |
+| branch | :white_check_mark: | - | Always set this to `${{ github.head_ref }}` in order to work both with pull requests and push events |
```

Comment:
I wonder, is it possible to remove this option altogether? I don't see a use case where one would like to push to a different branch then the one they checked out.